spec/features/scanned_resource_manifest_spec.rb

```diff
@@ -1,46 +0,0 @@
-require 'rails_helper'
```

Comment:
The login service checks obviously need to be removed, but can't we keep the status code checks to make sure auth is being enforced?